Libraries/src/Amazon.Lambda.RuntimeSupport/ExceptionHandling/JsonExceptionWriterHelpers.cs

```diff
@@ -22,96 +22,16 @@ internal class JsonExceptionWriterHelpers
         /// <summary>
         /// This method escapes a string for use as a JSON string value.
         /// It was adapted from the PutString method in the LitJson.JsonWriter class.
-        ///
-        /// TODO: rewrite the *JsonExceptionWriter classes to use a JSON library instead of building strings directly.
         /// </summary>
         /// <param name="str"></param>
         public static string EscapeStringForJson(string str)
         {
             if (str == null)
                 return null;
 
-            int n = str.Length;
-            var sb = new StringBuilder(n * 2);
-            for (int i = 0; i < n; i++)
-            {
-                char c = str[i];
-                switch (c)
-                {
-                    case '\n':
-                        sb.Append(@"\n");
-                        break;
+            string escapedString = JsonSerializer.Serialize(str);
```

Comment:
I was testing the output of this method with the one that already exists. They seem to differ.
For instance, with System.Text.Json, we use \u0027 to represent a single quote `'`. With the original code, we don't.

Input example which leads to different output results: `"  {\r\n    _id: '{{objectId()}}',\r\n    index: '{{index()}}',\r\n    guid: '{{guid()}}',\r\n    isActive: '{{bool()}}',\r\n    balance: '{{floating(1000, 4000, 2, \"$0,0.00\")}}',\r\n    picture: 'http://placehold.it/32x32',\r\n    age: '{{integer(20, 40)}}',\r\n    eyeColor: '{{random(\"blue\", \"brown\", \"green\")}}',\r\n    name: '{{firstName()}} {{surname()}}',\r\n    gender: '{{gender()}}',\r\n    company: '{{company().toUpperCase()}}',\r\n    email: '{{email()}}',\r\n    phone: '+1 {{phone()}}',\r\n    address: '{{integer(100, 999)}} {{street()}}, {{city()}}, {{state()}}, {{integer(100, 10000)}}',\r\n    about: '{{lorem(1, \"paragraphs\")}}',\r\n    registered: '{{date(new Date(2014, 0, 1), new Date(), \"YYYY-MM-ddThh:mm:ss Z\")}}',\r\n    latitude: '{{floating(-90.000001, 90)}}',\r\n    longitude: '{{floating(-180.000001, 180)}}',\r\n    tags: [\r\n      '{{repeat(7)}}',\r\n      '{{lorem(1, \"words\")}}'\r\n    ],  "`

Reply:
However, if we use LitJson. Code example below
`// Create a JsonData object to hold the string
JsonData jsonData = new JsonData(str);

// Serialize the JsonData object to a JSON string
string litjsonString = JsonMapper.ToJson(jsonData);
litjsonString = litjsonString.Trim('"').Trim();`

We get the same result